www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -829,6 +770,58 @@ function (declare, dom, style, fx, Toggler, on, query, Memory, Observable,
         }
       }));
 
+      //--- Bug review status ---//
+
+      var reviewStatusDialog = new ChangeReviewStatusDialog({
+        title      : 'Change review status',
+        message    : that.reportData.reviewComment
+                   ? that.reportData.reviewComment
+                   : 'Unreviewed',
+        reportData : this.reportData
+      });
+
+      var reviewStatusOptions = [];
+      for (var key in ReviewStatus) {
+        var value = ReviewStatus[key];
+        if (value !== ReviewStatus.UNREVIEWED ||
+            that.reportData.reviewStatus === ReviewStatus.UNREVIEWED)
+          reviewStatusOptions.push({
+            label : '<span class="customIcon review-status-'
+                  + util.reviewStatusCssClass(value)
+                  + '"></span>' + util.reviewStatusFromCodeToString(value),
+            value : value
+          });
+      }
+
+      var reviewStatusSelector = new Select({
+        class     : 'review-status-options',
+        options   : reviewStatusOptions,
+        value     : that.reportData.reviewStatus,
+        prevValue : that.reportData.reviewStatus,
+        onChange  : function () {
+          reviewStatusDialog.show();
+        },
+        reset     : function () {
+          this.set('value', this.prevValue, false);
+        }
+      });
+
+      reviewStatusDialog.set('reviewStatusSelector', reviewStatusSelector);
+
+      this.addChild(reviewStatusSelector);
+
+      //--- Review status comment message. ---//
+
+      var reviewComment = dom.create('span', {
```

Comment:
I like this idea very much! :yellow_heart:

However, I don't really like the fact that this icon is there even if no custom comment is given -- and by default it shows the same text as the review status drop-down next to it. Perhaps we could hide it, if the isn't a comment.